labman/db/testing.py

```diff
@@ -34,3 +36,15 @@ def setUpClass(cls):
             "https://localhost:21174", client_id, client_secret,
             server_cert=labman_settings.qiita_server_cert)
         qclient.post("/apitest/reset/")
+        # The above call only resets the qiita schema. Reset here the labman
```

Comment:
it was my understanding that we would be joining the labman (Bob) schema in to the qiita db schema.  making them one single db (and schema).  is this just for testing currently?